examples/wavenet/generate.py

```diff
@@ -58,11 +64,10 @@
     args.model, decoder, 'updater/model:main/predictor/decoder/')
 
 # Non-autoregressive generate
-if use_gpu:
-    x = chainer.cuda.to_gpu(x, device=args.gpu)
-    condition = chainer.cuda.to_gpu(condition, device=args.gpu)
-    encoder.to_gpu(device=args.gpu)
-    decoder.to_gpu(device=args.gpu)
+x = device.send(x)
+condition.device.send(x)
```

Comment:
Isn't `condition` a `numpy.ndarray` array here?

Reply:
Sorry, it seems I forgot to push the latest revision.